script/compile

```diff
@@ -64,6 +64,16 @@ arch()
 		cc --version | sed -n 1p
 		make -j"$JOBS" $V ALSA=1 psgplay
 		;;
+	macos-homebrew-gcc-portaudio)
+		HOMEBREW_GCC="$(find -L "$(brew --prefix)"/bin -name 'gcc-[0-9]*' 2>/dev/null | sort -Vr | head -n1)"
```

Comment:
Can we refactor the `find` and `if [ -z "$HOMEBREW_GCC" ] ...` things into a function such as `which_homebrew_gcc`?

Reply:
And so have something like `local HOMEBREW_GCC="$(which_homebrew_gcc)"` here?